org.OpenT2T.Sample.SuperPopular.Lamp/Philips Hue Light Bulb/ts/test.js

```diff
@@ -0,0 +1,41 @@
+var translator = require('./thingTranslator.js');
+var q = require('q');
```

Comment:
I was hoping the test file could also be TypeScript, so we could show direct consumers of this translator how to call it from typescript (e.g. if somebody is writing a bot or website in ts they could refer to the test.ts file to see how to call it). Any chance you could update this file to ts?